src/Objs/_defaultPageAttributes.js

```diff
@@ -1,6 +1,6 @@
 const defaultPageAttributes = {
   body: ["widgetlist", { only: "SectionWidget" }],
-  navigationBackgroundImage: ["reference", { only: ["Image"] }],
+  navigationBackgroundImage: ["reference", { only: ["Image", "Video"] }],
```

Comment:
* [ ] The VideoWidget also allows for `Download`. Can we go for consistency?

See https://github.com/Scrivito/scrivito_example_app_js/pull/167/files#diff-6c166dbb21e8fb0e3d946ba385826646R5